ui/app/components/ui/toggle-button/toggle-button.component.js

```diff
@@ -60,7 +60,10 @@ const ToggleButton = (props) => {
         thumbAnimateRange={[3, 18]}
         colors={colors}
       />
-      <div className="toggle-button__status-label">{ value ? onLabel : offLabel }</div>
+      <div className="toggle-button__status-container">
+        <span className="toggle-button__status-label toggle-button__status-label--off">{offLabel}</span>
```

Comment:
I could probably greatly reduce the complexity of this by...
1. `__status-container` -> `__status`
2. `__status-label __status-label--off` -> `__off-label` (and `__on-label`) respectively

in this way the modifier is only at the top level instead of there being two modifiers. it's definitely incorrect now and needs to change. Let me know your thoughts as you review.

Reply:
That suggestion seems pretty reasonable to me